src/components/styles/homePage/homePage.module.css

```diff
@@ -106,3 +106,14 @@
   bottom: 1.5em;
   box-shadow: 2px 8px 8px 2px rgba(0, 0, 0, 0.5);
 }
+
+.acceptReturnFab {
+  display: flex;
+  margin: 0;
+  padding: 0.8em ;
+  border-radius: 100px ;
+  position: fixed;
+  left: 2em;
+  bottom: 1.5em;
```

Comment:
Komuś odjebało konkretnie z tym użyciem `em` i to chyba byłem ja XD popraw to na `px` prosze zarówno w `acceptReturnFab` jak i w `addBookToLibraryFab`

Reply:
WIDZIAŁEM TO WŁAŚNIE I MOWIE CIEKAWE CZY ZWRÓCI UWAGĘ XD